tests/influence/test_influences.py

```diff
@@ -77,6 +78,13 @@ def analytical_linear_influences(
     return result
 
 
+@pytest.fixture(scope="function")
+def deterministic_seed():
```

Comment:
We already have [2 fixtures](https://github.com/appliedAI-Initiative/pyDVL/blob/develop/tests/conftest.py#L61-L76) that are used to seed random numbe generators.
Perhaps they can be combined (at least `pytorch_seed` and `deterministic_seed`?

Reply:
We could add torch.use_deterministic_algorithms to pytorch_seed. This should work then. What do you think?